src/element/textElement.test.ts

```diff
@@ -332,6 +332,12 @@ describe("Test getDefaultLineHeight", () => {
     //@ts-ignore
     expect(getDefaultLineHeight()).toBe(1.25);
   });
+
+  it("should return line height using default font family when passing an unknown font", () => {
+    const UNKNOWN_FONT = -1;
```

Comment:
```suggestion
    const UNKNOWN_FONT = 5;
```
This is just for simplicity so its a positive key and not to be confused with negative index when checking test.